apps/voting/contracts/Voting.sol

```diff
@@ -422,8 +580,73 @@ contract Voting is IForwarder, AragonApp {
         emit ExecuteVote(_voteId);
     }
 
+    /**
+     * @dev internal function to add the `_voter` to the list of delegated voters for the `_delegate` and update the delegate for the `_voter`
+     * @param _delegate address of the delegate
+     * @param _voter address of the voter
+     */
+    function _addDelegatedAddressFor(address _delegate, address _voter) internal {
+        uint256 delegatedVotersCount = delegatedVoters[_delegate].addresses.length;
+        require(delegatedVotersCount <= UINT_96_MAX, ERROR_MAX_DELEGATED_VOTERS_REACHED);
+
+        delegatedVoters[_delegate].addresses.push(_voter);
+        delegates[_voter] = Delegate(_delegate, uint96(delegatedVotersCount));
+        emit SetDelegate(_voter, _delegate);
+    }
+
+    /**
+     * @dev internal function to remove  the `_voter` from the list of delegated voters for the `_delegate`
+     * @param _delegate address of the delegate
+     * @param _voter address of the voter
+     */
+    function _removeDelegatedAddressFor(address _delegate, address _voter) internal {
+        uint256 delegatedVotersCount = delegatedVoters[_delegate].addresses.length;
+        require(delegatedVotersCount > 0, ERROR_DELEGATE_NOT_SET);
```

Comment:
Excess check - never triggers

Reply:
next assert actually checking more strict condition

Reply:
Fixed in 8ec459c5b194b304aabf010a046c97f8eae259b6.